apps/sms/services/test/unit/mock_moz_mobile_connections.js

```diff
@@ -0,0 +1,24 @@
+/*exported MockMozMobileConnections */
+
+// The real navigator.mozMobileConnections is not a real array
+'use strict';
+
+var MockMozMobileConnections = {
+  0: {
+    iccId: 'SIM 1',
+    addEventListener: sinon.stub(),
```

Comment:
__question:__ IIRC we usually don't use sinon in mocks? Maybe it's better to use plain empty functions here and stub in tests?